code/client/src/components/NFTGrid.jsx

```diff
@@ -73,22 +71,31 @@ const NFTGridItem = ({ disabled, style, styleFullView, imageWrapperStyle, imageW
   return (
     <GridItem style={fullView ? styleFullView : style} hoverable={false} onClick={() => !fullView && interactable && setFullView(true)} data-full-view={fullView}>
       {!fullView &&
-        <Row style={{ height: wrapperStyle.height || 'auto' }}>
-          <Image
-            preview={false} src={util.replaceIPFSLink(metadata?.image) || FallbackImage} fallback={FallbackImage}
-            wrapperStyle={wrapperStyle} style={{ objectFit: 'cover', width: '100%', height: '100%' }}
-          />
+        <Row>
```

Comment:
need wrapperStyle to keep the heights uniform for all items in the grid across multiple rows

Reply:
Added the wrapper style back.